src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/_help.py

```diff
@@ -426,6 +426,38 @@
         az functionapp plan create -g MyResourceGroup -n MyPlan --number-of-workers 4 --sku S1
```

Comment:
please change to
`az functionapp plan create -g -n --min-instances 1 --max-burst 10 --sku EP1

Reply:
Ok, I didn't update these yet because it's still in preview. You think it would be ok to have help show commands that may not work for certain subs

Reply:
I'm not sure on what will work for what subs and where - but I do know currently what we have scheduled to go out in doc form on Apr 3 is the following CLI command:
```bash
az functionapp plan create -g <resource-group> -n <plan-name> -l <region> --number-of-workers 1 --sku EP1
```

Just let me know if / when this will break to use more optimal naming as @ahmedelnably suggested so I can make sure to update appropriate docs when the az version rolls out